packages/react/src/__tests__/ReactElementJSX-test.js

```diff
@@ -282,9 +282,12 @@ describe('ReactElement.jsx', () => {
       expect(() =>
         ReactDOM.render(JSXRuntime.jsx(Parent, {}), container),
       ).toErrorDev(
-        'Warning: React.jsx: Spreading a key to JSX is a deprecated pattern. ' +
-          'Explicitly pass a key after spreading props in your JSX call. ' +
-          'E.g. <Child {...props} key={key} />',
+        'Warning: An props object containing a "key" prop is being spread into JSX:\n' +
+          '  let props = {key: ...};\n' +
```

Comment:
Let's do `key: someKey` to avoid using `...` both as syntax and a placeholder?

Reply:
It's a non-trivial change in the code but I'll try.

Reply:
Well, maybe it doesn't make sense. Because it still writes out the other properties with `...`.

Reply:
I put the key in the front even if that's not the regular order and kept `...` for other props.